src/view/site-generes-view.js

```diff
@@ -0,0 +1,35 @@
+
+
+const generToggle = (array) => {
+  let checker;
+  if(array.length>1){
+    checker ='Genres';}
+  else{
+    checker ='Genre';
+
+  }
+
+
+  return checker;
+};
```

Comment:
Эту функцию тоже полезно переписать в одну строку. Так же удобнее читать, верно?
```suggestion
const generToggle = (array) => (array.length>1),'Genres':'Genre';
```